### PR TITLE
Musark 444

### PR DESCRIPTION
**Pull request by rituvesh**

### CODE REVIEW

File: src/reducers/move/index.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ export const moveNode = (nodeId, destination, doneBy, callback) => {
     items: [].concat(nodeId)
   }
   return {
-    types: [MOVE_OBJECT, MOVE_OBJECT_SUCCESS, MOVE_OBJECT_FAILURE],
```
Comment (jarlah) [inline src/reducers/move/index.js:45]: 👍

File: src/reducers/picklist/index.js [hunk 2]
```diff
@@ -53,11 +60,44 @@ const clearItems = (type) => (state) => ({
   [type]: []
 });
 
+export const getPath = (pathStr) => {
+  const pathStrArr = pathStr.substr(1, pathStr.length - 2).split(',')
+  // EX: ,1,2,3,19, will be transformed to 1,2,3,19
```
Comment (jarlah) [inline src/reducers/picklist/index.js]: we need to correct this comment to:
 // EX: ,1,2,3,19, will be transformed to 1,2,3,

File: src/reducers/picklist/__tests__/PicklistReducer-test.js [hunk 3]
```diff
@@ -482,4 +483,71 @@ describe('PicklistReducer', () => {
...
+          path: [],
+          value: node3
+        }
+      ],
+      OBJECT: DEFAULT_OBJECTS
+    }
+    assert.deepStrictEqual(JSON.stringify(expectedState), JSON.stringify(newState), 'Should update path for node 1')
+  })
+
+it('Refresh node updates path works', () => {
```
Comment (jarlah) [inline src/reducers/picklist/__tests__/PicklistReducer-test.js]: this test is either not doing what it should, or anything it all. I am not able to see what this test is testing.

File: src/reducers/picklist/__tests__/PicklistReducer-test.js [hunk 4]
```diff
@@ -482,4 +483,71 @@ describe('PicklistReducer', () => {
...
+    result:{
+      path: ',1,2,3,',
+      pathNames: [{ id: 1},{ id: 2},{ id: 3}] },
+    id: 1
+  })
+  const expectedState =  {
+    NODE: [
+      {
+        marked: false,
+        path: [null],
```
Comment (jarlah) [inline src/reducers/picklist/__tests__/PicklistReducer-test.js]: this seems to be a bug for me. why is path expected to be array with one null element when you modify the state with a result of path with path and pathnames?

File: src/reducers/picklist/__tests__/PicklistReducer-test.js [hunk 5]
```diff
@@ -482,4 +483,71 @@ describe('PicklistReducer', () => {
...
+    }
+    assert.deepStrictEqual(JSON.stringify(expectedState), JSON.stringify(newState), 'Should update path for node 1')
+  })
+
+it('Refresh node updates path works', () => {
+  const newState = picklistReducer(testState, {
+    type: LOAD_ONE_NODE_SUCCESS,
+    result:{
+      path: ',1,2,3,',
+      pathNames: [{ id: 1},{ id: 2},{ id: 3}] },
```
Comment (jarlah) [inline src/reducers/picklist/__tests__/PicklistReducer-test.js]: Formatting is important. bring the last semicolon on this line down.

**Review by jarlah — CHANGES_REQUESTED** [review]

File: src/reducers/picklist/index.js (hunk 2 above)
Comment (rituvesh) [reply src/reducers/picklist/index.js]: Done!

File: src/reducers/picklist/__tests__/PicklistReducer-test.js (hunk 4 above)
Comment (rituvesh) [reply src/reducers/picklist/__tests__/PicklistReducer-test.js]: Done with description!

**Review by jarlah — APPROVED** [review]

**Pull request merged**